pubspec.yaml

```diff
@@ -49,7 +49,13 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  test: any  
+  test: any
+  flutter_launcher_icons: ^0.7.4
```

Comment:
Where is this package being used?

Reply:
The flutter launcher icon is used to set launcher images on iOS app. It does not have any effect on app size as it is `dev_dependencies` and not `dependencies`